<launch>

    <!-- Launch the simulation world -->
    <include file="$(find tiago_iaslab_simulation)/launch/start_simulation.launch">
        <arg name="world_name" value="iaslab_assignment2"/>
    </include>

    <!-- Launch the AprilTag -->
    <include file="$(find tiago_iaslab_simulation)/launch/apriltag2.launch" />

    <!-- Launch the navigation stack -->
    <include file="$(find tiago_iaslab_simulation)/launch/navigation.launch" />

    <!-- Run the straight line node -->
    <node pkg="tiago_iaslab_simulation" type="get_straightline_node" name="get_straightline_node" output="screen"/>

</launch>